### DISCUSSION TITLE
Is there a way to show result in L/(100km)?

### DISCUSSION THREAD

**Discussion by alexchesn** [Qalculate/qalculate-gtk#497] [Q&A] [answered]
Is there a way to show Fuel Economy in L/(100km)? I don't see a dedicated unit and I can't figure out a way to show it properly through conversion.
I see Table C.47. Units: Fuel Economy: `mpg` which is helpful, but this is the only unit available.

<img width="765" alt="Screenshot 2023-10-20 at 15 41 22" src="https://github.com/Qalculate/qalculate-gtk/assets/50532889/19daa9df-ba25-4cb7-b66c-edee64521249">

**Comment by hanna-kn** [comment, marked as answer]
Conversion to l/100km has now been fixed in libqalculate (https://github.com/Qalculate/libqalculate/commit/c1f7cdd72fde6cd530de19d3443ff476247ff4d5).

Without the fix, convert to `cl/km`, `1/100 l/km`, or `0.01 l/km`, deactivate _Mode->Unit Display->Place Units Separately_, or add a new unit.

**Comment by alexchesn** [comment]
Thanks, @hanna-kn. Deactivate Mode->Unit Display->Place Units Separately worked for me. 
Also, I would like to thank you for your hard work. It is truly amazing project! I'm a big fun for many years.
